f5lbaasdriver/v2/bigip/driver_v2.py

```diff
@@ -167,10 +167,18 @@ def _call_rpc(self, context, loadbalancer, entity, api_dict,
             agent_host, service = self._setup_crud(
                 context, loadbalancer, entity, **kwargs)
             rpc_callable = getattr(self.driver.agent_rpc, rpc_method)
-            rpc_callable(
-                context, api_dict, service, agent_host,
-                **kwargs
-            )
+
+            the_port = kwargs.get("the_port_id", None)
+            LOG.info(the_port)
+            if the_port:
+                LOG.info('the_port is not None')
+                rpc_callable(
+                    context, api_dict, service,
+                    agent_host, the_port_id=the_port
```

Comment:
Rpc method does not define kwargs. Does it works?

Reply:
look at the create_member() in rpc , there is one case .